core/templates/dev/head/css/oppia.css

```diff
@@ -65,6 +65,10 @@ button.md-button.md-default-theme.oppia-learner-continue-button:hover {
   background-color: #115FD4;
 }
 
+/* Attributes for material-icons. */
+.material-icons{
```

Comment:
(If it is, please check all the other instances of .material-icons visually to make sure there aren't any regressions, and remove any duplicate "vertical-align: middle;" attributes.)